pkg/ctl/scheme.go

```diff
@@ -20,16 +20,17 @@ limitations under the License.
 package ctl
 
 import (
+	acmeinstall "github.com/jetstack/cert-manager/pkg/internal/apis/acme/install"
+	cminstall "github.com/jetstack/cert-manager/pkg/internal/apis/certmanager/install"
+	metainstall "github.com/jetstack/cert-manager/pkg/internal/apis/meta/install"
+	corev1 "k8s.io/api/core/v1"
 	metainternalversion "k8s.io/apimachinery/pkg/apis/meta/internalversion"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	kscheme "k8s.io/client-go/kubernetes/scheme"
-
-	acmeinstall "github.com/jetstack/cert-manager/pkg/internal/apis/acme/install"
-	cminstall "github.com/jetstack/cert-manager/pkg/internal/apis/certmanager/install"
-	metainstall "github.com/jetstack/cert-manager/pkg/internal/apis/meta/install"
```

Comment:
Sorry to be a pain, but why did the cert-manager imports move to the top?
I thought convention was to put them in a separate block at the bottom.

Reply:
Fixed